chord/chord.go

```diff
@@ -55,18 +57,21 @@ type ChordNode struct {
 	stubsPool stubsPoolWithMux
 
 	// for gRPC server functionality
-	chordServer *grpc.Server
+	grpcServer *grpc.Server
+
+	chunkSize int
```

Comment:
I would still want to separate chord functionality, e.g., lookups, from other functionality of our program such as downloading files, etc. 
However, I have not come up with an API yet, so for the current API we have, we need to move chunkSize inside ChordConfig struct, which you can see in my PR.